### PR TITLE
[IOTDB-2881] add author serialize and deserialize and add serialize and deserialize test

### PR DESCRIPTION
**Pull request by RYH61**
add author serialize and deserialize
add serialize and deserialize test
Use the AuthorizerManager to call login() in the openSession method of the SessionManager

### CODE REVIEW

File: server/src/main/java/org/apache/iotdb/db/mpp/sql/planner/plan/PlanFragment.java [hunk 1]
```diff
@@ -94,12 +96,14 @@ private PlanNode getPlanNodeById(PlanNode root, PlanNodeId nodeId) {
     return null;
   }
 
-  public static PlanFragment deserialize(ByteBuffer byteBuffer) throws IllegalPathException {
+  public static PlanFragment deserialize(ByteBuffer byteBuffer)
+      throws IllegalPathException, AuthException {
```
Comment (wangchao316) [inline server/src/main/java/org/apache/iotdb/db/mpp/sql/planner/plan/PlanFragment.java:100]: why does add AuthException  in deserialize mothod?

Comment (RYH61) [reply server/src/main/java/org/apache/iotdb/db/mpp/sql/planner/plan/PlanFragment.java]: I have made changes to remove the AuthorException

**Review by wangchao316 — APPROVED** [review]
LGTM

**Review by mychaow — APPROVED** [review]
LGTM

**Pull request merged**